next.config.mjs

```diff
@@ -37,6 +37,13 @@ const nextConfig = {
     locales: ["en"],
     defaultLocale: "en",
   },
+  webpackDevMiddleware: (config) => {
+    config.watchOptions = {
+      poll: 1000,
+      aggregateTimeout: 300,
+    };
+    return config;
+  },
```

Comment:
Please share the docs for `webpackDevMiddleware`

Reply:
Sorry for the late reply; the fix was sourced from stackoverflow. I will have a look at this over the weekend and get back :)

Reply:
Thank you 👍 

I worry when things come from stackoverflow as they are often out of date.

Please check and let us know

Reply:
It has been 2 weeks with no response, so I will remove these changes and merge the docker changes